Marlin/Marlin_main.cpp

```diff
@@ -8489,7 +8545,7 @@ void process_next_command() {
         gcode_G28();
         break;
 
-      #if PLANNER_LEVELING
+      #if PLANNER_LEVELING && !ENABLED(AUTO_BED_LEVELING_UBL) || ENABLED(AUTO_BED_LEVELING_UBL) && ENABLED(UBL_MESH_EDIT_ENABLED)
```

Comment:
This has been patched as `#if PLANNER_LEVELING || ENABLED(AUTO_BED_LEVELING_UBL)` and that's good.